shared/getMetrics.js

```diff
@@ -262,8 +262,8 @@ const getMetrics = ({
         testsCount,
         testsFailedCount,
         unexpectedBehaviorCount,
-        highImpactPassedAssertionCount,
-        highImpactFailedAssertionCount,
+        severeImpactPassedAssertionCount,
+        severeImpactFailedAssertionCount,
         moderateImpactPassedAssertionCount,
         moderateImpactFailedAssertionCount,
         commandsCount,
```

Comment:
Since metrics are saved in the database does that mean we should have a migration to make sure the metrics are up-to-date?

Reply:
Since this hasn't yet gone to the production environment yet to affect that database, no need to write that migration. There wouldn't be any metrics that include those properties just yet.